src/CollapsibleCard/index.js

```diff
@@ -0,0 +1,157 @@
+      <div>
+        <h4
+          className={cc([
+            !isDisabled ? "title" : "title--disabled",
+            "fontWeight--bold",
+            "fontSize--l",
+            "padding--top--l",
+            "padding--left--l",
+            "fontFamily--body",
+          ])}
+        >
+          {title}
+        </h4>
+        <div
+          className={cc([
+            !isDisabled ? "subtitle" : "subtitle--disabled",
+            "padding--left--l",
+            "padding--bottom--l",
+            "margin--top--xxs",
+          ])}
+        >
+          {subtitle}
+        </div>
+      </div>
+      {helperText && (
+        <div style={{ display: "flex" }}>
+          <div
+            className="collapsible-card--helperText padding--right--l fontSize--s"
+            style={{ alignSelf: "center" }}
+          >
+            {helperText}
+          </div>
+        </div>
+      )}
+    </div>
+  );
+
+  return isOpen && !isDisabled ? (
+    <div
+      className={cc([
+        "collapsible-card--content-card",
+        {
+          "content-card--error": hasError,
+          "content-card--hover": !disableHover && hover,
+        },
+        "rounded--bottom",
+        "bgColor--white",
+      ])}
+    >
+      <div
+        className="collapsible-card--title-expanded"
+        role="button"
+        tabIndex={0}
+        onKeyUp={({ key }) => {
+          if (key === "Enter") onTitleContainerClick(false, "close");
+        }}
+        aria-expanded="true"
+        onClick={() => onTitleContainerClick(false, "close")}
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+      >
+        {titleContainerJSX}
+      </div>
+      <div className="padding--all--l">{children}</div>
+    </div>
+  ) : (
+    <div
+      className={cc([
+        "collapsible-card--content-card",
+        {
+          "content-card--error": hasError,
+          "content-card--disabled": isDisabled,
+          "content-card--hover": !disableHover && hover,
+        },
+        "content-card--closed",
+        "rounded--all",
+        "bgColor--white",
+      ])}
+      role="button"
+      tabIndex={0}
+      onKeyUp={({ key }) => {
+        if (key === "Enter") onTitleContainerClick(isDisabled, "open");
+      }}
+      aria-expanded="false"
+      aria-disabled={isDisabled ? "true" : "false"}
+      onClick={() => onTitleContainerClick(isDisabled, "open")}
+    >
+      {titleContainerJSX}
+    </div>
+  );
+};
+
+CollapsibleCard.propTypes = {
+  /** Accepts any content as children */
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node),
+  ]).isRequired,
+  /** Card title */
+  title: PropTypes.string,
+  /** Card subtitle */
+  subtitle: PropTypes.string,
+  /** Card helper text, placed on the right side of the title container */
+  helperText: PropTypes.node,
+  /** Controls whether card is opened */
+  isOpen: PropTypes.bool.isRequired,
```

Comment:
**Suggestion**

The `isOpen` prop is already defaulting to `false` in the component definition, so we could make this optional (also, undefined is falsy)


```
const CollapsibleCard = ({
  title,
  subtitle,
  helperText,
  isOpen = false, // <------------
  onOpen = () => {},
  onClose = () => {},
  isDisabled = false,
  onDisabledClick = () => {},
  hasError = false,
  disableHover = false,
  children,
}) => {
```